frontend/src/components/common/SimpleTable.tsx

```diff
@@ -294,9 +295,9 @@ export default function SimpleTable(props: SimpleTableProps) {
     return <Loader title={t('Loading table data')} />;
   }
 
-  let filteredData = displayData;
+  let filteredData = displayData!;
   if (filterFunction) {
-    filteredData = displayData.filter(filterFunction);
+    filteredData = displayData!.filter(filterFunction);
```

Comment:
Weird, typescript was normally smart enough to understand this is prevalidated, IIRC.

Reply:
there's an additional case where it can be an undefined, but we only check for null. I'll remove undefined from displayData type